modules/roles/client/components/role-delete.component.ts

```diff
@@ -0,0 +1,48 @@
+import {
+	Component,
+	OnInit
+}
+	from '@angular/core';
+
+import {
+	Role
+} from '../models/role';
+
+import {
+	Router,
+	ActivatedRoute,
+	Params
+} from '@angular/router';
+
+/* Import role service */
+import {
+	RoleService
+} from '../services/roles.service';
+
+
+@Component({
+	selector: 'delete-role',
+	providers: [RoleService],
+	templateUrl: '../views/role-delete.html'
```

Comment:
we'll probably want to import the roles service at the roles module level (in roles.module.ts) we can sort of treat it as a singleton that way